docs/user/production-considerations/task-manager-health-monitoring.asciidoc

```diff
@@ -90,6 +90,15 @@ a| Runtime
 
 | This section tracks excution performance of Task Manager, tracking task _drift_, worker _load_, and execution stats broken down by type, including duration and execution results.
 
+a| Capacity Estimation
+
+| This section provides a rough estimate about the sufficiency of its capacity. As the name suggests, these are estimates based on historical data and should not be used as predictions. We recommend using these estimations when following the Task Manager <<task-manager-scaling-guidance>>.
```

Comment:
```suggestion
| This section provides a rough estimate about the sufficiency of its capacity. As the name suggests, these are estimates based on historical data and should not be used as predictions. Use these estimations when following the Task Manager <<task-manager-scaling-guidance>>.
```

Reply:
In general, we try to avoid using the word "we", which I see throughout these docs.  In many places, you can replease with "you".  When making a recommendation, just tell the reader what to do:

We recommend using these estimations when following the Task Manager scaling guidance > Use these estimates when following the Task Manager scaling guidance.